README.md

```diff
@@ -205,16 +259,29 @@ The following is merely on overview over the most important parameters.
 
 ## Example Call
 
-* Change the basic stuff like inputbase and outputbase directories
-set RUN_FEATURE_COUNTS_DEXSEQ, RUN_RNASEQC, RUN_KALLISTO, RUN_ARRIBA, runFingerprinting to FALSE.
-* Check if your sample type is in possibleTumorSampleNamePrefixes
-G Set your star index (GENOME_STAR_INDEX) and gene models (GENE_MODELS) parameters to what your created for (1) and (2).
-* Set GENOME_GATK_INDEX to point to the new FASTA file (1), for which you have created a dict file. The dict file should be in the same directory as the FASTA.
+* Change the basic stuff like `inputbase` and `outputbase` directories
+set `RUN_FEATURE_COUNTS_DEXSEQ`, `RUN_RNASEQC`, `RUN_KALLISTO`, `RUN_ARRIBA`, `runFingerprinting` to `FALSE`.
+* Check if your sample type is in `possibleTumorSampleNamePrefixes`
+* Set your star index (`GENOME_STAR_INDEX`) and gene models (`GENE_MODELS`) parameters to what your created for (1) and (2).
```

Comment:
Typo `your created` -> `you created`.

Reply:
> sense/antisense

You mean because the expression data itself, dependent on the lab protocol, may yield reads in sense or antisense direction, or not? Hm ...

Maybe "mapping strand" and "reverse-mapping strand" would fit better then.